diff -up HandBrake-40ec4ebaa2958b2549125dd525b6fcb2bd595d69/libhb/internal.h.cl HandBrake-40ec4ebaa2958b2549125dd525b6fcb2bd595d69/libhb/internal.h
--- HandBrake-40ec4ebaa2958b2549125dd525b6fcb2bd595d69/libhb/internal.h.cl	2017-01-20 11:51:28.511218948 +0100
+++ HandBrake-40ec4ebaa2958b2549125dd525b6fcb2bd595d69/libhb/internal.h	2017-01-20 11:51:43.974305794 +0100
@@ -8,7 +8,7 @@
  */
 
 #include "hbffmpeg.h"
-#include "extras/cl.h"
+#include <CL/cl.h>
 #ifdef USE_QSV
 #include "qsv_libav.h"
 #endif
