diff -up HandBrake-1.7.3/libhb/module.defs.orig HandBrake-1.7.3/libhb/module.defs
--- HandBrake-1.7.3/libhb/module.defs.orig	2024-05-15 23:50:54.273783155 +0200
+++ HandBrake-1.7.3/libhb/module.defs	2024-05-16 00:07:33.709692520 +0200
@@ -71,8 +71,12 @@ else
         LIBHB.libxml2 := $(shell $(PKGCONFIG.exe) --silence-errors --cflags libxml-2.0)
     endif
     ifneq (,$(LIBHB.libxml2))
-        LIBHB.libxml2 := $(patsubst -I%,%,$(LIBHB.libxml2))
-        LIBHB.GCC.I += $(LIBHB.libxml2)
+        LIBHB.libxml2.I := $(filter -I%,$(LIBHB.libxml2))
+        LIBHB.libxml2.I := $(patsubst -I%,%,$(LIBHB.libxml2.I))
+        LIBHB.GCC.I += $(LIBHB.libxml2.I)
+        LIBHB.libxml2.D := $(filter -D%,$(LIBHB.libxml2))
+        LIBHB.libxml2.D := $(patsubst -D%,%,$(LIBHB.libxml2.D))
+        LIBHB.GCC.D += $(LIBHB.libxml2.D)
     else
         LIBHB.GCC.I += /usr/include/libxml2
     endif
