diff --git a/libhb/muxavformat.c b/libhb/muxavformat.c
index 929e0dd0cfe9..f1032bd9f841 100644
--- a/libhb/muxavformat.c
+++ b/libhb/muxavformat.c
@@ -7,6 +7,7 @@
    For full terms see the file COPYING file or visit http://www.gnu.org/licenses/gpl-2.0.html
  */
 
+#include <time.h>
 #include <ogg/ogg.h>
 #include "libavcodec/bsf.h"
 #include "libavformat/avformat.h"
diff --git a/libhb/work.c b/libhb/work.c
index 3c4dee193bf5..9bf128bdbc73 100644
--- a/libhb/work.c
+++ b/libhb/work.c
@@ -7,6 +7,7 @@
    For full terms see the file COPYING file or visit http://www.gnu.org/licenses/gpl-2.0.html
  */
 
+#include <time.h>
 #include "handbrake/handbrake.h"
 #include "libavformat/avformat.h"
 #include "handbrake/decomb.h"
diff -up HandBrake-1.6.1/libhb/qsv_common.c.orig HandBrake-1.6.1/libhb/qsv_common.c
--- HandBrake-1.6.1/libhb/qsv_common.c.orig	2023-01-22 17:36:49.000000000 +0100
+++ HandBrake-1.6.1/libhb/qsv_common.c	2024-03-11 14:28:19.626425759 +0100
@@ -2183,7 +2183,9 @@ static int hb_qsv_parse_options(hb_job_t
             }
             else if (!strcasecmp(key, "async-depth"))
             {
-                int async_depth = hb_qsv_atoi(value, &err);
+                char *str = hb_value_get_string_xform(value);
+                int async_depth = hb_qsv_atoi(str, &err);
+                free(str);
                 if (!err)
                 {
                     job->qsv.async_depth = async_depth;
